### PR TITLE
needed override of com.thoughtworks.xstream version

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

**Comment by mbiarnes** [comment]
Jenkins execute full downstream build

File: pom.xml [hunk 1]
```diff
@@ -265,6 +265,9 @@
     <version.com.github.javaparser>3.13.10</version.com.github.javaparser>
 
     <version.org.apache.commons.csv>1.6</version.org.apache.commons.csv>
+
+    <!-- override needed for 7.4.0.CR2 -->
```
Comment (mareknovotny) [inline pom.xml]: please don't add such comments into upstream here ;)

**Comment by mareknovotny** [comment]
Jenkins execute full downstream build

File: pom.xml (hunk 1 above)
Comment (mbiarnes) [reply pom.xml]: when the full downstream build is green. before merging this I will remove this comment

**Comment by mareknovotny** [comment]
the failed tests now from 2nd run are different, so I would say it is ok.
Please send PR for master too.

**Comment by krisv** [comment]
@mbiarnes can you make sure the same gets fixed on master too (although with ip-bom getting removed it's going to be slightly different location I guess).

**Comment by mbiarnes** [comment]
@krisv already done: https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/pom.xml#L108

**Pull request merged**